satpy/tests/reader_tests/test_ahi_hrit.py

```diff
@@ -36,19 +36,40 @@ def _get_reader(self, mocked_init, mda, filename_info=None):
         HRITJMAFileHandler.mda = mda
         return HRITJMAFileHandler('filename', filename_info, {})
 
+    def _get_acq_time(self, nlines):
+        """Get sample header entry for scanline acquisition times.
+
+        Lines: 1, 21, 41, 61, ..., nlines
+        Times: 1970-01-01 00:00 + (1, 21, 41, 61, ..., nlines) hours
+
+        So the interpolated times are expected to be 1970-01-01 +
+        (1, 2, 3, 4, ..., nlines) hours. Note that there will be some
```

Comment:
The use of hours here is confusing me, it sounds like we add integer hours.

Reply:
Good catch, it's adding integer seconds